libc-bottom-half/crt/crt1-command.c

```diff
@@ -3,8 +3,20 @@ extern void __wasm_call_ctors(void);
 extern int __main_void(void);
 extern void __wasm_call_dtors(void);
 
+// Commands should only be called once per instance. This simple check ensures
+// that the `_start` function isn't started more than once. `address_space(1)`
+// tells clang to put this variable in a wasm global.
```

Comment:
Does that actually work?  It seems a little risky to depend on this now.  Why not just is a simple static?

Reply:
No big reason; and depending on it breaks LLVM 11, which we still support for now, so I'll switch to a static.